tools/pipelines-tasks/azure-pipelines/ci-build.yml

```diff
@@ -29,9 +29,22 @@ schedules:
 pool:
   name: 'msix-packaging-pool'
 
+# Set the Azure variables to run AVD publish tests on required azure subscription service connection containing AVD resources.
+# Not setting these variables will leave running AVD publish test cases and will not fail the pipeline
```

Comment:
AVD publish test case is made optional so that if user raising the PR (or running the pipeline) do not have azure or AVD resources should still be able see the CI pipeline failure. In other words, say if some one is making changes to just the MSIX or VHDX generation tasks and not AVD publish, they should not be blocked as they do not have an AVD account.